docs/serverless-v2/02-01-serverless-v2.md

```diff
@@ -2,27 +2,25 @@
 title: Architecture
 ---
 
-Serverless v2 relies on [Knative Serving](https://knative.dev/docs/serving/) for deploying and managing functions, and [Tekton Pipelines](https://github.com/tektoncd/pipeline) for creating Docker images. See how these and other resources process a lambda within a Kyma cluster:
+Serverless v2 relies on [Knative Serving](https://knative.dev/docs/serving/) for deploying and managing functions and [Kubernetes Jobs](https://kubernetes.io/docs/concepts/workloads/controllers/jobs-run-to-completion/) for creating Docker images. See how these and other resources process a lambda within a Kyma cluster:
 
-![Serverless v2 architecture](./assets/serverless-v2-architecture.svg)
+![Serverless architecture](./assets/serverless-architecture.svg)
```

Comment:
Hm... before the creation of Job there is a step that creates ConfigMap - it is described in Details topic as a separate section so this diagram should be aligned with the next sections.